test/e2e/mongodbtests/mongodbtests.go

```diff
@@ -340,6 +340,27 @@ func Connect(mdb *mdbv1.MongoDBCommunity, opts *options.ClientOptions) error {
 	})
 }
 
+// Connect performs a connectivity check by initializing a mongo client
+// with the SRV connection string and inserting a document into the MongoDB
+// resource. Custom client options can be passed, for example to configure TLS.
+func ConnectSRV(mdb *mdbv1.MongoDBCommunity, opts *options.ClientOptions) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Minute)
+	defer cancel()
+	mongoClient, err := mongo.Connect(ctx, opts.ApplyURI(mdb.MongoSRVURI()))
+	if err != nil {
+		return err
+	}
+
+	return wait.Poll(time.Second*1, time.Second*30, func() (done bool, err error) {
```

Comment:
[tiny nit] I think we never used named return values both in Community and Enterprise. I am not against it, maybe it's worth a discussion? Our coding style document is quite out of date and probably not often referred to...